src/mote/modules/find.py

```diff
@@ -26,24 +26,25 @@
 import urllib.parse as ulpr
 from datetime import datetime
 
-directory_path = os.path.dirname("/srv/web/meetbot/")
-recognition_pattern = "(.*)[\-\.]([0-9]{4}-[0-9]{2}-[0-9]{2})-([0-9]{2}\.[0-9]{2})"
+from flask import current_app as app
 
 
 def find_meetings_by_substring(search_string: str):
     """
     Return list of meetings returned from search
     """
+    directory_path = os.path.dirname(app.config["MEETING_DIR"])
+    recognition_pattern = "(.*)[\-\.]([0-9]{4}-[0-9]{2}-[0-9]{2})-([0-9]{2}\.[0-9]{2})"
```

Comment:
You don't need to move that one.
Note that I see this same variable at three different places already. Maybe we can make it a configuration item too?